valve/gamemodes/llhl.cfg

```diff
@@ -0,0 +1,117 @@
+//Llhl
```

Comment:
I think it would be more better to capitalize the gamemode name

Reply:
Done